examples/cms-sanity/lib/sanity.js

```diff
@@ -0,0 +1,14 @@
+import sanityClient from '@sanity/client'
+
+const options = {
+  // Find your project ID and dataset in `sanity.json` in your studio project
+  dataset: 'production',
+  projectId: process.env.SANITY_PROJECT_ID,
+  useCdn: process.env.NODE_ENV === 'production',
+  // useCdn == true gives fast, cheap responses using a globally distributed cache.
+  // Set this to false if your application require the freshest possible
+  // data always (potentially slightly slower and a bit more expensive).
```

Comment:
To do preview, we need to generate and use API token:

https://www.sanity.io/docs/preview-content-on-site

Reply:
Added